composer.json

```diff
@@ -92,7 +92,8 @@
     },
     "conflict": {
         "doctrine/common": "<2.7",
-        "doctrine/mongodb-odm": "<2.0"
+        "doctrine/mongodb-odm": "<2.0",
+        "doctrine/persistence": ">1.3"
```

Comment:
```suggestion
        "doctrine/persistence": "<1.3"
```

Reply:
Ok got it !

Reply:
Actually changing the namespace of Doctrine/Common/Persistence to Doctrine/Persistence will break if `"doctrine/persistence": "<2.0"`.

Reply:
But if I do so, I have this error:

 ```
composer install
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Warning: The lock file is not up to date with the latest changes in composer.json. You may be getting outdated dependencies. It is recommended that you run `composer update` or `composer update <package name>`.
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - api-platform/core 2.5.x-dev conflicts with doctrine/persistence[1.3.0].
    - doctrine/persistence 1.3.0 conflicts with api-platform/core[2.5.x-dev].
    - Installation request for api-platform/core 2.5.x-dev -> satisfiable by api-platform/core[2.5.x-dev].
    - Installation request for doctrine/persistence 1.3.0 -> satisfiable by doctrine/persistence[1.3.0].
```

Because of this, I'll stick to what you said.

Reply:
To avoid doctrine/persistence 2.0 I added 
```
"conflict": {        
  "doctrine/common": ">=3.0"
}
```
in my project. Now i can install the api platform. Could this work or do I miss something?